Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,117 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
+  then
+    git_URL=""
+    git_SHA=""
+    ruby_URL="https://github.com/xu-cheng/homebrew-portable/releases/download/20160512/portable-ruby-2.0.0-p648.x86_64-darwin9.tar.gz"
```

Comment:
Would be good to move before this PR is merged. 32-bit should be used on 10.5 as I believe it should still work up to 10.11 and will work on more systems (as you don't seem to be checking for 32-bit support here).